infra/nnfw/cmake/options/options_aarch64-android.cmake

```diff
@@ -3,8 +3,8 @@
 # NOTE BUILD_ANDROID_TFLITE(JNI lib) is disabled due to BuiltinOpResolver issue.
 # tensorflow-lite does not build BuiltinOpResolver but JNI lib need it
 # Related Issue : #1403
-option(BUILD_ANDROID_TFLITE "Enable android support for TensorFlow Lite" ON)
-option(BUILD_ANDROID_BENCHMARK_APP "Enable Android Benchmark App" ON)
+option(BUILD_ANDROID_TFLITE "Enable android support for TensorFlow Lite" OFF)
+option(BUILD_ANDROID_BENCHMARK_APP "Enable Android Benchmark App" OFF)
```

Comment:
@chunseoklee Is there any reason to disable `BUILD_ANDROID_BENCHMARK_APP` ?

Reply:
@hseok-oh It was a mistake. updated.